retrofit/WebServices/src/main/java/com/example/aperez/retrofittest/mvp/model/ResponseModel.java

```diff
@@ -30,7 +32,11 @@ public ResponseModel(Bus bus) {
     public void getLatestImages(){
         Retrofit retrofit = new Retrofit.Builder()
```

Comment:
As a good practice. You could create another class, to manage the Retrofit services. For example, Services. Not necessary to apply change, only a sugerency